client/store/products.js

```diff
@@ -69,6 +79,8 @@ export default function productReducer (state=initialState, action) {
             return { ...state, products: state.products.map(product => product.id === action.product.id ? action.product : product) }
         case DELETE_PRODUCT:
             return { ...state, products: state.products.filter(product => product.id !== action.product.id) }
+        case FETCH_ORDERS:
+            return { ...state, userOrders : action.userOrders }
```

Comment:
If you guys want me to move the cart/order stuff into a separate file in the store, i can do that later